app/lib/server/functions/setRealName.js

```diff
@@ -20,7 +20,11 @@ export const _setRealName = function(userId, name) {
 	}
```

Comment:
On line 10 `name` will be converted from `undefined` to `''` and on line 18 the comparison will be false executing the following updates even when not setting the name